bmi_index.js

```diff
@@ -0,0 +1,28 @@
+const formElement = document.querySelector(".bmi-form");
+
+const heightInput = document.querySelector("#height");
+const weightInput = document.querySelector("#weight");
+
+const bmiMessage = document.querySelector(".result");
+
+// const button = document.querySelector("button");
+
+// function calculate(heightInput, weightInput) {
+//   let result = heightInput.value / weightInput.value ** 2;
+//   console.log(result);
+// }
```

Comment:
Commented code ;-) You don't want that in your production branch ;-)

Reply:
Removed it! Thx !